hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/LogEventViewGenerator.java

```diff
@@ -21,6 +22,8 @@
   private static final Logger LOG = LoggerFactory.getLogger(LogEventViewGenerator.class);
   private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
 
+  private static final List<String> SUMMARY_KEYS = List.of("message", "body", "event");
```

Comment:
nit: can we put reference links, why are those keys consider? https://github.com/opentracing/specification/blob/master/semantic_conventions.md#log-fields-table
https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/logs/v1/logs.proto#L108